src/assets/css/global.css

```
@@ -151,12 +151,14 @@ textarea:focus {
   display: flex;
   align-items: center;
   justify-content: center;
+  flex-wrap: wrap;
 }
 
 .flex-items a {
   display: block;
-  margin: 0 20px;
+  margin: 20px;
   transition: all 0.4s ease-in-out; 
+  width: 350px;
```

```suggestion
  width: 20vw;
```
I think 20vw could be good so if we have 3 columns ( 3 * 20 = 60 %) and a loot of whitespace 😄 
But you should definitely try this out. maybe this will not looks great on mobile...

```suggestion
  width: 21rem;
```

350px should be something around 21 to 22 rem maybe this is the better solution for this

---

A) I was giving width in **px** because we won't have to set it at different breakpoints.
B) But if we give it a relative value, we would have to change it's value at different breakpoints too.

Which one would you go with? @schmelto

---

I am a fan of using rem :)